### DISCUSSION TITLE
v4.2 RC1

### DISCUSSION THREAD

**Discussion by RussKie** [gitextensions/gitextensions#11252] [General]
### Release Notes Highlights
Requires: **.NET 6.0 Desktop Runtime [v6.0.22](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-6.0.22-windows-x64-installer)** or later

* Git 2.41 is recommended
* Various performance improvements
* Various UI improvements
* Improvements to Git Commandlog dialog
* Improvements to the Rebase dialog
* Allow "Save as..." multiple files
* Editor can now move line up/down with <kbd>ALT + UP</kbd> and <kbd>ALT + DOWN</kbd>
* OpenSSH to use SSH_ASKPASS if a password is requested (requires OpenSSH 8.4 or higher; no effect for older versions of OpenSSH)
* Auto-set GE as editor only in current environment
* More user friendly handling for "Could not load file or assembly" exceptions
* JIRA plugin now supports personal access tokens
* Vertical tab (<kbd>SHIFT + ENTER</kbd>) is now treated as a line feed
* Add support to build GE for Windows on Arm64 (WoA). This, however, requires manual build.

Full change log: https://github.com/gitextensions/gitextensions/blob/v4.2-rc1/GitUI/Resources/ChangeLog.md#version-42-21-oct-2023

![Downloads](https://img.shields.io/github/downloads/gitextensions/gitextensions/v4.2-rc1/total.svg)

<hr /><em>This discussion was created from the release <a href='https://github.com/gitextensions/gitextensions/releases/tag/v4.2-rc1'>v4.2 RC1</a>.</em>

**Comment by chirontt** [comment]
A manual build of this v4.2 RC1 version for Windows on Arm64 is available in my fork: https://github.com/chirontt/gitextensions/releases/tag/v4.2-rc1-win32-aarch64
